astropy/io/tests/mixin_columns.py

```diff
@@ -0,0 +1,137 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Not 100% sure it is best to have a new file. Since ECSV covers everything, this could also stay at the top of `test_ecsv.py`.